modules/OCPP201/device_model/everest_device_model_storage.cpp

```diff
@@ -130,12 +204,35 @@ EverestDeviceModelStorage::EverestDeviceModelStorage(
         }
     }
 
+    // build components from EVerest config
+    // This is our mapping strategy:
+    // Component.name = module_type
+    // Component.instance = module_id
+    // Component.evse.id/connector = mapping of module
+    // FIXME: what to do when there are different mappings from module & impl?
+    for (const auto& [module_id_type, module_config] : this->module_configs) {
+        ComponentKey component_key;
+        component_key.name = module_id_type.module_type;
+        component_key.instance = module_id_type.module_id;
+        const auto& mapping = this->mappings.at(module_id_type.module_id);
+        if (mapping.module.has_value()) {
+            const auto& module_mapping = mapping.module.value();
+            component_key.evse_id = module_mapping.evse;
```

Comment:
we have to be careful here to not map to id `0`. EVerest allows it, OCPP2.0.1 doesnt.

![image](https://github.com/user-attachments/assets/cd73d20b-84f5-4a26-b7af-a3c2b76617b6)